__tests__/html/fluentTheme/side-by-side.wide.html

```diff
@@ -424,29 +424,6 @@
 [7]: cite:1 \"Meet the Contoso Team\"`,
           "type": "message"
         }
-      ], [
```

Comment:
A little sad we don't have an "all-integrated" look for blueprint state, but given context may not be bad either

Reply:
Yeah... after 7 years, unsurprisingly, I still didn't come up with a good way to organize tests.

Layout vs. behavior? Adding as much variables as possible for permutation? No ideas.